### DISCUSSION TITLE
the way to store the latest chain snapshot?

### DISCUSSION THREAD

**Discussion by Reskorey** [filecoin-project/lotus#5717] [General Q&A] [answered]
I always have a queustion,Why not store latest chain snapshot files by IPFS node or Filecion miner node, the data is slow to download on Amazon server in China?

**Comment by rjan90** [comment]
If you already are running lotus daemon, and are synced, you can prune it yourself with `lotus chain export`, and then import the pruned chain you created. But I agree, better availability of the latest chain snapshot would be nice.

**Comment by LexLuthr** [comment, marked as answer]
https://lotu.sh/docs/set-up/nodes-in-china/
